scaling_performance/optimizing_storage.adoc

```diff
@@ -222,7 +222,7 @@ storage technology), such as DeviceMapper and OverlayFS. Each has advantages
 and disadvantages.
 
 For more information about OverlayFS, including supportability and usage caveats, see the
-link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/?version=7[Red Hat Enterprise Linux (RHEL) 7 Release Notes] for your version.
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.5_release_notes/technology_previews_file_systems[Red Hat Enterprise Linux (RHEL) 7 Release Notes] for your version.
```

Comment:
Verify that something in common-attributes isn't right for "Red Hat Enterprise Linux (RHEL)".